ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/claimantresponse/CCDResponseRejection.java

```diff
@@ -1,23 +1,21 @@
 package uk.gov.hmcts.cmc.ccd.domain.claimantresponse;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
 import lombok.Builder;
+import lombok.EqualsAndHashCode;
 import lombok.Value;
 import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
-
```

Comment:
Please rearrange using (Ctrl+Option+O) imports here. This line is logical separation of imports.

Reply:
when i do (Ctrl+Option+O) .. it groups 

import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Value;
import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;

The logic behind Lombok imports being grouped with our domain model imports is beyond me.